subscription-engine/kafka-ws/index.js

```diff
@@ -5,6 +5,11 @@ const util = require('util');
 
 const { ConfigManager, Logger } = require('@dojot/microservice-sdk');
 
+const application = require('./app/App');
+const websocketTarball = require('./app/WebsocketTarball');
+const terminus = require('./app/Terminus');
```

Comment:
I'm confused if this is going to be done in graceful shutdown PR, but just to remember...

With the adoption of `StateManager`, it seems to me that this is no longer necessary:
`const terminus = require('./app/Terminus');`

Remove from `package.json` too...

Reply:
Ok, I'll remove it in the graceful shutdown PR.

Reply:
Yes make sense, I'll add it.